### ISSUE TITLE
Cannot close the screen saver options sheet

### ISSUE THREAD

**Issue by nsayer** [JohnCoates/Aerial#495]
The "Done" button on the options sheet does not close the sheet. There is no workaround I can discover other than force-quitting the preferences app. This is under Mojave.

**Comment by staticdreams** [comment]
Same issue. Mojave Beta 11

**Comment by brinary001** [comment]
Yup!  Same here.  Running most recent Mojave beta on 2017 15" MBP

**Comment by dash16** [comment]
Same here, Mojove public release.

**Comment by EerkHof** [comment]
Same here too. Mojave official release
